### PR TITLE
Add variables

### PR DESCRIPTION
**Pull request by angehung5**
1. input_variables.txt is updated by adding surface roughness length (Z0) and changing to southeast domain
2. domain settings (nlat, nlon) and reading section in canopy_driver.f90 are changed for the updated list

### CODE REVIEW

**Comment by drnimbusrain** [comment]
@angehung5 This looks good, and I am close to merging with develop, but why are the Monin-Obukhov Lengths (MOL) always zero in the new input_variables.txt file?

File: canopy_driver.F90 [hunk 1]
```diff
@@ -97,6 +97,14 @@ program canopy_driver
...
            real(rk)    :: fh           !forest/canopy height
            real(rk)    :: ws           !wind speed at reference height above canopy (10 m)
+           real(rk)    :: clu          !clumping index
+           real(rk)    :: lai          !leaf area index
+           real(rk)    :: vtype        !vegetation type
+           real(rk)    :: ffrac        !forest fraction
+           real(rk)    :: ust          !friction velocity (u*)
+           real(rk)    :: csz          !cosine of solar zenith angle
+           real(rk)    :: z0           !surface roughness length
+           real(rk)    :: mol          !Monin-Obukhov length
```
Comment (drnimbusrain) [inline canopy_driver.F90:107]: @angehung5 All values for MOL are zero here.

**Review by drnimbusrain — CHANGES_REQUESTED** [review]
@angehung5 Please update MOL to have values other than zero.

**Comment by angehung5** [comment]
File updated. Do I need to pull a request again or you can see the change?

**Comment by zmoon** [comment]
> File updated. Do I need to pull a request again or you can see the change?

We see it. Changes you make in the branch associated with the pull request automatically show up here.

**Review by drnimbusrain — CHANGES_REQUESTED** [review]
@angehung5 Any idea why the magnitude of values for MOL are so low (typically range from +/- 1 to 200 m)?

**Comment by zmoon** [comment]
<details><summary>That is consistent with the Stull figure for "typical" conditions</summary>

image

</details>

**Comment by angehung5** [comment]
OK, I found the problem. I made a mistake in the heat capacity of air. Now the MOL values should be more reasonable.

**Review by drnimbusrain — APPROVED** [review]

**Pull request merged**